OutOfSchool/OutOfSchool.EmailSender/SendGridAccessibilityService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Logging;
+
+using Quartz;
+
+namespace OutOfSchool.EmailSender;
+
+public interface ISendGridAccessibilityService 
+{
+    bool IsSendGridAccessible { get; set; }
+}
+public class SendGridAccessibilityService : ISendGridAccessibilityService
+{
+    public bool IsSendGridAccessible { get; set; }
+}
+
+public class EmailSenderJobListener : IJobListener
```

Comment:
(Code style) Too many different clases in one file. Please either split the file or join them all into 1 class.